participants/check_package_built_at_source.py

```diff
@@ -90,12 +90,12 @@ def quality_check(self, action, wid):
                     if status != "succeeded":
                         msg.append("Build status is '%s' for %s (against %s)." %
                                 (status, source_build, build_against))
-                        if status != "excluded":
+                        if status not in ["excluded", "disabled"]:
                             result = False
 
-        if targets:
-            msg.append("Not build against %s" % ", ".join(targets))
-            result = False
+        #if targets:
+        #    msg.append("Not build against %s" % ", ".join(targets))
+        #    result = False
```

Comment:
This however, completely disables check_package_built_at_source making it a noop.  That said, if you look at the rest of the code it doesn't at all do what it claims to do. Entire logic needs to be rewritten.